evm/contracts/ChainlinkClient.sol

```diff
@@ -22,6 +23,7 @@ contract ChainlinkClient {
   uint256 constant private ARGS_VERSION = 1;
   bytes32 constant private ENS_TOKEN_SUBNAME = keccak256("link");
   bytes32 constant private ENS_ORACLE_SUBNAME = keccak256("oracle");
+  address constant private CHAINLINK_REGISTRY = 0xFf4C35dE072E83a9495008d9082AE080a8a96465;
```

Comment:
The ChainlinkRegistry contract is deployed to each test network at the same address:
- [Ropsten](https://ropsten.etherscan.io/address/0xFf4C35dE072E83a9495008d9082AE080a8a96465#code)
- [Rinkeby](https://rinkeby.etherscan.io/address/0xFf4C35dE072E83a9495008d9082AE080a8a96465#code)
- [Kovan](https://kovan.etherscan.io/address/0xFf4C35dE072E83a9495008d9082AE080a8a96465#code)

This constant can be changed in the future when we come up with a better way to manage the key that deployed the contract.